src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -88,7 +94,7 @@ class CrashHeader extends React.Component {
     const {title, beforeTitle, hideGuide, stackView, stackType, newestFirst} = this.props;
 
     let titleNode = (
-      <h3 className="pull-left">
+      <h3 className="pull-left" style={{marginBottom: 0}}>
```

Comment:
We should have a "Heading component' with all h1, h2, h3.... Since I didn't want to change this code too much, I found it easier to add an inline style